fanuc_r2000ib_support/urdf/r2000ib165f_macro.xacro

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+  <xacro:include filename="$(find fanuc_resources)/urdf/common_materials.xacro"/>
+  
+  <xacro:macro name="fanuc_r2000ib165f" params="prefix">
+    <!-- links -->
+    <link name="${prefix}base_link">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/visual/base_link.stl"/>
+        </geometry>
+        <xacro:material_fanuc_yellow />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/collision/base_link.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_1">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/visual/link_1.stl"/>
+        </geometry>
+        <xacro:material_fanuc_yellow />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>	  
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/collision/link_1.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_2">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/visual/link_2.stl"/>
+        </geometry>
+        <xacro:material_fanuc_white />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://fanuc_r2000ib_support/meshes/r2000ib165f/collision/link_2.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_3">
+      <!--       The manual and also the model files show this link in the 90deg position (ie right angle) -->
+      <!--       However, the coordinate system treats the straight configuration as 0deg and the limits as asymmetric -->
```

Comment:
Could you explain this comment a little more?

Reply:
No ... I rather remove it as this (and the mitsubishi launch files) were copied from a not so well matching template

Reply:
Perhaps you could compare/use the `fanuc_r2000ic_support` package I pushed [to my fork](https://github.com/ros-industrial/fanuc_experimental/compare/indigo-devel...gavanderhoorn:r2000ic)?